request.js

```diff
@@ -116,15 +116,15 @@
         return callbacks;
     };
 
-    exports['get'] = function (url, query) {
+    exports.get = function (url, query) {
```

Comment:
Check with @maxigimenez . I remember making this change when I first used the script and he told me there was a reason for this.